src/ledger/LedgerManagerImpl.cpp

```diff
@@ -1164,6 +1164,10 @@ LedgerManagerImpl::applyTransactions(
     }
 
     prefetchTransactionData(txs);
+    auto applyTimer =
+        LogSlowExecution{"LedgerManagerImpl::applyTransactions",
```

Comment:
I don't think we should be adding performance information by default: it's getting too noisy.
Maybe we need to rethink how the "perf" partition works.

I sketched a change in https://github.com/MonsieurNicolas/stellar-core/commit/ab4be49baee4db2782d7a2cf3c2d282820c421a3 that may be closer to what we want.
Reason I sketched it was that I was getting too many `fetch-XYZ` messages even though my log level was `INFO`.
I didn't have time to find the right way to log (so ended up logging with the `INFO` level even though it's not).